src/borg/archiver.py

```diff
@@ -2797,10 +2808,10 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
 
             if sort_by:
                 sort_by_default = 'timestamp'
-                filters_group.add_argument('--sort-by', metavar='KEYS', dest='sort_by',
-                                           type=SortBySpec, default=sort_by_default,
-                                           help='Comma-separated list of sorting keys; valid keys are: {}; default is: {}'
-                                           .format(', '.join(AI_HUMAN_SORT_KEYS), sort_by_default))
+                filters_group.add_argument('--sort-by', metavar='KEYS', dest='sort_by', type=SortBySpec,
+                    default=sort_by_default,
+                    help='Comma-separated list of sorting keys; valid keys are: {}; default is: {}'
+                         .format(', '.join(AI_HUMAN_SORT_KEYS), sort_by_default))
```

Comment:
shall we just increase to 160 or so to not have to reformat such stuff?

Reply:
Before I made these changes there were lines longer than 200! The problem, at least in my setup, is that past 120, you can't do side-by-side diffs without scrolling back and forth to see the whole line. Even the github diffs on this page (shown top/bottom, not side-by-side) only show about 96 characters.  I think we can still make it pretty at 120, and in cases where it really is best to be longer, we can always '# noqa' or '# noqa: E501' the line or file.

Reply:
ok